### DISCUSSION TITLE
Is vrcx down?

### DISCUSSION THREAD

**Discussion by AverageRindo** [vrcx-team/VRCX#641] [General]
When i tried to launch vrcx last night it came up to a blank screen, even after uninstalling and reinstalling it nothing changed and i have the same blank white screen on the application.
![Screenshot 2023-08-31 150206](https://github.com/vrcx-team/VRCX/assets/143749722/2381cc76-9c66-430b-b5f1-0babba51ab3b)

**Comment by Myrkie** [comment]
VRCX is not a website, it cannot be "down" you have encountered a application error follow troubleshooting steps at 
https://github.com/vrcx-team/VRCX/wiki#vrcx-opens-to-a-white-screen

**Reply by AverageRindo** [reply]
renaming the sqlite file worked for me, thanks for the help
